src/deepsparse/yolo/pipelines.py

```diff
@@ -209,11 +234,12 @@ def _preprocess_image(self, image) -> numpy.ndarray:
             image = cv2.imread(image)
 
         image = self._make_channels_last(image)
+        original_image_shape = image.shape[:2]
```

Comment:
At this point, image can be 4D and thus you would get `original_image_shape` (B, H, W) not (H,W). Maybe let's move it a bit lower in the logic?

Reply:
I think you meant we'll get (B, H) That's a great callout, infact moving it lower also won't fix the problem, I'll add in a if condition for this